app/views/thesis/deduplicate.html.erb

```diff
@@ -6,40 +6,32 @@
 
 <link href="https://cdn.datatables.net/1.10.24/css/jquery.dataTables.min.css" rel="stylesheet">
 
-<h3 class="title title-page">Theses with co-author(s)</h3>
-
-<%= render 'shared/you_are' %>
-
-<%= form_tag(thesis_deduplicate_path, method: "get") do %>
-  <label>
-    Show only theses from: 
-    <select name="graduation">
-      <option value="all">All terms</option>
-      <% @terms.each do |term| %>
-        <option value="<%= term %>"<%= ' selected="selected"'.html_safe if params[:graduation].to_s == term.to_s %>>
-          <%= term.in_time_zone('Eastern Time (US & Canada)').strftime('%b %Y') %>
-        </option>
-      <% end %>
-    </select>    
-  </label>
-
-  <%= submit_tag('Apply filter', class: 'btn button-primary') %>
-<% end %>
-
-<table class="table" id="thesisQueue" title="Thesis processing queue">
-  <thead>
-    <tr>
-      <th scope="col">Title</th>
-      <th scope="col">Author(s)</th>
-      <th scope="col">Coauthor(s)</th>
-      <th scope="col">Department(s)</th>
-      <th scope="col">Degree date</th>
-    </tr>
-  </thead>
-  <tbody>
-    <%= render(partial: 'thesis/duplicate_thesis', collection: @thesis) || render('select_empty') %>
-  </tbody>
-</table>
+<div class="layout-3q1q layout-band">
```

Comment:
We may want to consider moving the report menu (eventually) to allow for full width reports. For now this is fine but want us to be considering other options since we have unfortunately set a max width in our core styles so there is no way to get a "wide view" of these reports and taking up a portion with a menu is unfortunate.

Reply:
Yeah, I agree that this is unfortunate. I didn't feel like this was the place to solve a fly-out submenu (which is the only alternative I can think of at the moment, aside from a wholesale design system update).

If you like, I can make a ticket in the backlog to revisit the submenu layout with some UX assistance.

Reply:
I've made ticket ETD-442 for consideration of this question.